packages/modules/web-viewer-react/src/services/RpcInitializer.ts

```diff
@@ -0,0 +1,81 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import { BentleyCloudRpcManager } from "@itwin/core-common";
+
+import type { RegisterClientOptions } from "../types";
+
+/**
+ * The RpcInitializer handles registration of backends/instantiates RpcInterface clients.
+ */
+export class RpcInitializer {
+  static readonly orchestratorUrl = `https://${globalThis.IMJS_URL_PREFIX}api.bentley.com`;
+
+  /**
+   * Instantiates a `DefaultBackend` and optional `CustomBackend`s. The `DefaultBackend`
+   * must have at least one Rpc Interface in the `DefaultBackend.rpcInterfaces` array.
+   * @param options: RegisterClientOptions
+   * @returns void
+   */
+  public static registerClients(options: RegisterClientOptions) {
+    const formattedOptions = this.formatDefaultBackendOptions(options);
+
+    BentleyCloudRpcManager.initializeClient(
+      formattedOptions,
+      options.defaultBackend.rpcInterfaces
+    );
+
+    if (!options?.customBackends) {
+      return;
+    }
+
+    for (const { config, rpcInterfaces } of options.customBackends) {
+      BentleyCloudRpcManager.initializeClient(config, rpcInterfaces);
+    }
+  }
+
+  private static formatDefaultBackendOptions(options: RegisterClientOptions) {
```

Comment:
feel like this can be cleaned up & condensed, but i think it depends on what we end up doing for default

Reply:
Simplified